omnigibson/object_states/aabb.py

```diff
@@ -11,7 +11,7 @@ def _get_value(self):
             aabb_low, aabb_hi = BoundingBoxAPI.compute_aabb(self.obj)
             aabb_low, aabb_hi = np.array(aabb_low), np.array(aabb_hi)
         elif self.obj.prim_type == PrimType.CLOTH:
-            particle_positions = self.obj.root_link.particle_positions
+            particle_positions = self.obj.root_link.get_particle_positions(keypoints_only=False)
```

Comment:
Should we keep the original `particle_positions` property and add a new property called `subsampled_particle_positions`? I wonder whether it would be cleaner this way.

Reply:
done!